cumulusci/tasks/release_notes/task.py

```diff
@@ -13,6 +13,39 @@
 )
 
 
+class AllGithubReleaseNotes(BaseGithubTask):
+
+    task_options = {
+        "repos": {
+            "description": (
+                "The list of owner, repo key pairs to generate release notes for." + " Ex: 'owner': SalesforceFoundation \
+                                                                                           'repo': 'NPSP'"
+            ),
+            "required": False,
+        },
+
+    }
+
+    def _run_task(self):
+        table_of_contents = "<h1>Table Of Contents</h1><ul>"
+        body = ""
+        for project in self.options["repos"]:
+            print(f"Owner: {project['owner']}")
+            print(f"Repo: {project['repo']}")
+            if project["owner"] and project["repo"]:
+                release = self.github.repository(project["owner"],project["repo"]).latest_release().body
+                table_of_contents += f"<a href=#{project['repo']}><li><slot name={project['repo']} />{project['repo']}</li></a>"
+                release_project_header = f"<h1 slot id={project['repo']} name={project['repo']}>{project['repo']}</h1><br>"
+                release_html = self.github.markdown(release,mode="gfm",context="{}/{}".format(project["owner"],project["repo"]))
+                body += f"{release_project_header}{release_html}"
+        table_of_contents += "</ul>"
+        
+        result = table_of_contents + body
+        result = result.replace("\n","<br>") # fixing newline issue
+        with open("github_release_notes.html", "w") as f:
```

Comment:
It doesn't look like we're generating a complete HTML file.

Reply:
Do you mind expanding upon that?

Reply:
The output is a document fragment - it has no `<html>`, `<head>`, or `<body>` element.

It's possible that that isn't wrong for a specific use case, but it's making me a little concerned about shipping this task to the public because we will have to explain and support it. I'm wondering whether this should be kept internal to support the SFDO docs team.

Reply:
The table of contents needs to go inside the `<body>` tag. I would suggest also adding a `<head>` section with the title set to something like "Release Notes".